CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
+## 2.1.38 "TBD" - October 17, 2022
+<!---
+Packaged by Joseph Makar <arthurk@sentinelone.com> on Oct 17, 2022 12:31 -0800
+--->
+
+Kubernetes:
+* Fix bug that caused logging of the Kubernetes cache stats to agent status. 
+
+Other
+* Add to agent's status its generation and average generation time. 
```

Comment:
Might want to reword it to say something along the line of "Added support for --debug flag to the scalyr-agent-2 status command. When this flag is used, agent prints additional debug related information with the status output. NOTE: Right now it's only supported with the healthcheck config option".

Reply:
Thanks, copy-pasted it :)